.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,52 @@
+name: "Code scanning - action"
+
+# runs scanning on every push, every pull request, and on Saturdays at 10pm.
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 22 * * 6'
```

Comment:
Are you telling me I can run this every Saturday for the next 5 years and it will cost me/us $0 United States Dollars?

If not, when does this cron job start costing money?